opentech/apply/review/templates/review/review_list.html

```diff
@@ -0,0 +1,22 @@
+{% extends "base-apply.html" %}
+{% load bleach_tags %}
+
+{% block content %}
+<div class="wrapper wrapper--breakout wrapper--admin">
+    <div class="wrapper wrapper--medium">
+        <h2 class="heading heading--no-margin">Reviews</h2>
+        <h5>For <em>{{ submission.title }}</em></h5>
+    </div>
+</div>
+
+<table>
+{% for question, answers in review_data.items %}
+    <tr>
+        <th>{{ question }}</th>
+        {% for answer in answers %}
+            <td>{{ answer|bleach }}</td>
```

Comment:
the answers seem to contain some boolean values, leading to an `argument cannot be of 'bool' type, must be of text type` TypeError

Reply:
fixed in 60ead48